migrations/sql/08_add_agent_type_pg_stat_monitor.down.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE `metrics` MODIFY COLUMN `agent_type` Enum8(
+    'qan-agent-type-invalid'=0,
+    'qan-mysql-perfschema-agent'=1,
+    'qan-mysql-slowlog-agent'=2,
+    'qan-mongodb-profiler-agent'=3,
+    'qan-postgresql-pgstatements-agent'=4
```

Comment:
I am not sure, but maybe here missing qan-postgresql-pgstatmonitor-agent? Because in up file it is presented.

Reply:
it's `down.sql` file, it will be used if need to downgrade migrations.

Reply:
I see. I thought that it is something like on/off. Thanks.